packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -389,6 +406,13 @@ class SwipeableViews extends Component {
         passive: false,
       },
     );
+    this.mouseMoveListener = addEventListenerEnhanced(this.rootNode, 'mousemove', (event) => {
```

Comment:
If we don't need the listener to be passive, can we use the *idiomatic* way. I mean like `onMouseUp` for instance.